components/script/dom/htmldialogelement.rs

```diff
@@ -124,4 +125,25 @@ impl HTMLDialogElementMethods<crate::DomTypeHolder> for HTMLDialogElement {
             .dom_manipulation_task_source()
             .queue_simple_event(target, atom!("close"));
     }
+
+    // https://html.spec.whatwg.org/multipage/interactive-elements.html#dom-dialog-requestclose
+    fn RequestClose(&self, return_value: Option<DOMString>) {
+        let element = self.upcast::<Element>();
+        let target = self.upcast::<EventTarget>();
+
+        if !element
+            .has_attribute(&local_name!("open"))
+        {
+            return;
```

Comment:
Can you add spec comments for this function?

Something like
```
// Step 1. If thisdoes not have an open attribute, then return.
```
 with TODOs for missing steps